FindOpenmpt.cmake

```diff
@@ -9,6 +9,7 @@ find_package(PkgConfig)
 if(PKG_CONFIG_FOUND)
   pkg_check_modules (OPENMPT libopenmpt)
   list(APPEND OPENMPT_INCLUDE_DIRS ${OPENMPT_INCLUDEDIR})
+  link_directories(${OPENMPT_LIBRARY_DIRS})
```

Comment:
If you want to fix this CMake find module, then do it the right way by never trusting pkg-config only.

Reply:
I would prefer to remove pkg-config completely, as it always brought problems when installed in the OS and Kodi addon depends it includes.

As a result, a part was referenced to their own build and Lib link used the OS installed type.

Reply:
Some addons avoid this bug in a hidden, even stupid way: 
https://github.com/kodi-pvr/pvr.vuplus/blob/Matrix/FindTinyXML.cmake#L9-L12
The `find_package(PkgConfig)` is not defined and `PKG_CONFIG_FOUND` is false (not set)

Reply:
How you see the right way (have self no idea how it can be best)? e.g. like way on Kodi itself:
```cmake
find_package(PkgConfig)
if(PKG_CONFIG_FOUND)
  pkg_check_modules(PC_MODPLUG libmodplug QUIET)
endif()

find_path(MODPLUG_INCLUDE_DIRS libmodplug/modplug.h
                               PATHS ${PC_MODPLUG_INCLUDEDIR})
find_library(MODPLUG_LIBRARIES NAMES modplug
                               PATHS ${PC_MODPLUG_LIBDIR})

include(FindPackageHandleStandardArgs)
find_package_handle_standard_args(Modplug REQUIRED_VARS MODPLUG_INCLUDE_DIRS MODPLUG_LIBRARIES)

mark_as_advanced(MODPLUG_INCLUDE_DIRS MODPLUG_LIBRARIES)
```

As long the PkgConfig is used it bring me always (only the `${OPENMPT_LIBRARY_DIRS}` prevented them):
```
        linux-vdso.so.1 (0x00007fffef57e000)
WRONG>>>        libmodplug.so.1 => /lib/x86_64-linux-gnu/libmodplug.so.1 (0x00007fdbaeb9e000)
        libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fdbae9bc000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fdbae9a2000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fdbae7b7000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fdbae669000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fdbaef5f000)
```

Reply:
Seems to go now, the secret was "PC_" in the beginning. Without this, `MODPLUG_LIBRARIES` was not overwritten by` find_library`.

About before: Had not deleted at the beginning of the previously created cmake system and cmake update run has this is not changed.